assets/js/src/frontend/hgf.js

```diff
@@ -16,6 +16,20 @@ export const HFG = function () {
 	this.init();
 };
 
+const toggleAria = (elements, action) => {
+	elements.forEach(function (element) {
+		if (!element) {
+			return;
+		}
+		if (action === 'add') {
+			element.setAttribute('aria-hidden', 'true');
+		}
+		if (action === 'remove') {
+			element.removeAttribute('aria-hidden');
+		}
+	});
+};
```

Comment:
For some marginal filesize improvements, we can optimize this function to use a boolean for the action flag instead of a string. 

This way, when minified, the boolean will take fewer bytes than the string (the string is also used in the calls):

```suggestion
const toggleAria = (elements, add = true) => {
	elements.forEach(function (element) {
		if (!add) {
			element.removeAttribute('aria-hidden');

			return;
		}

		element.setAttribute('aria-hidden', 'true');
	});
};
```